consent/models.py

```diff
@@ -26,17 +28,35 @@ class DataSharingConsentQuerySet(models.query.QuerySet):
     except is not saved in the database until committed.
     """
 
-    def get(self, *args, **kwargs):
+    def proxied_get(self, *args, **kwargs):
```

Comment:
nit: i'm not sure about this name - it's not intuitively clear what it means, but i don't have any alternatives to suggest.